### PR TITLE
Labels do not becomes color anymore in figure options panel for qt toolb...

### PR DESCRIPTION
**Pull request by Acanthostega**
Filtering by labels is sufficient, not clean, but working.

Are you interested in making a rigid but clean widget to avoid such problems in the future ? If yes, I can probably do it in two or three weeks, when I will have sufficient time.

### CODE REVIEW

File: lib/matplotlib/backends/qt_editor/formlayout.py [hunk 1]
```diff
@@ -61,6 +61,8 @@
 
 import datetime
 
+BLACKLIST = ["Title", "Label"]
```
Comment (tacaswell) [inline lib/matplotlib/backends/qt_editor/formlayout.py]: I would make these all lower case

File: lib/matplotlib/backends/qt_editor/formlayout.py [hunk 2]
```diff
@@ -262,7 +264,7 @@ def setup(self):
                 continue
             elif tuple_to_qfont(value) is not None:
                 field = FontLayout(value, self)
-            elif is_color_like(value):
+            elif label not in BLACKLIST and is_color_like(value):
```
Comment (tacaswell) [inline lib/matplotlib/backends/qt_editor/formlayout.py]: and do `label.lower()` here just to be on the safe side.

File: lib/matplotlib/backends/qt_editor/formlayout.py (hunk 1 above)
Comment (tacaswell) [inline lib/matplotlib/backends/qt_editor/formlayout.py]: Can you also use a `set`, trivial performance gain...

**Comment by Acanthostega** [comment]
Ok, added the set improvement. Better if the blacklist increases one day ! But the label.lower() seems to cost a lot too. Maybe check against the "brute" label is better ?

**Comment by tacaswell** [comment]
The performance optimizations on the `set` was a bit of bike shedding which you indulged me in ;)

In most cases, it is simpler to normalize strings before checking them for being in the set rather than including all permutations of capitalization you want to support.

**Comment by tacaswell** [comment]
@Acanthostega Thank you!  

I think this is your first mpl contribution, congratulation!

**Pull request merged**